src/main/java/seedu/address/commons/core/Config.java

```diff
@@ -13,7 +13,7 @@
     public static final Path DEFAULT_CONFIG_FILE = Paths.get("config.json");
 
     // Config values customizable through config file
-    private String appTitle = "Address App";
+    private String appTitle = "Loanbook App";
```

Comment:
Why do we need the App after Loanbook?

Reply:
The name "Loanbook App" is simply a placeholder. Now that you mention it, it makes sense just to have it as "Loanbook".

Eventually, we will probably change the name again before the final submission.